pmoves/services/chat-relay/tests/test_relay.py

```
@@ -0,0 +1,347 @@
+    def test_content_field_priority(self):
+        """Content is extracted in correct priority order."""
+        # The code checks: content -> response -> message
+        from main import ChatRelayService, Config
+
+        config = Config(
+            nats_url="nats://localhost:4222",
+            supabase_url="http://localhost:3010",
+            supabase_service_role_key="test-key",
+            health_port=8102
+        )
+        service = ChatRelayService(config)
```

Ruff (0.14.8)

Local variable `service` is assigned to but never used

Remove assignment to unused variable `service`

(F841)